src/monocle_apptrace/exporters/aws/s3_exporter.py

```diff
@@ -110,7 +110,16 @@ def __serialize_spans(self, spans: Sequence[ReadableSpan]) -> str:
         try:
             # Serialize spans to JSON or any other format you prefer
             span_data_list = [span.to_json() for span in spans]
-            return "[" + ", ".join(span_data_list) + "]"
+            valid_json_list = []
+            for span_data in span_data_list:
```

Comment:
Can we just use the following?
 "\n".join(span_data_list) + "\n"

Reply:
this will  return the pretty json , but we are looking for inline json (ndjson).